23-freight-car-door-motion-and-location/config.json

```diff
@@ -0,0 +1,14 @@
+# uncomment set your own product UID and serial number
```

Comment:
Should the word "uncomment" be removed?

Reply:
Yeah, I had it there when the next line actually was commented out!